api/src/main/java/org/pmiops/workbench/auth/ProfileService.java

```diff
@@ -98,27 +95,6 @@ public Profile getProfile(User user, boolean checkEmailVerification) {
     profile.setInstitutionalAffiliations(user.getInstitutionalAffiliations()
         .stream().map(TO_CLIENT_INSTITUTIONAL_AFFILIATION)
         .collect(Collectors.toList()));
-    EmailVerificationStatus userEmailVerificationStatus = user.getEmailVerificationStatus();
-    // if verification is pending or unverified, need to query MailChimp and update DB accordingly
-    // Note: This is pretty slow, we shouldn't be calling it on every Profile lookup. Leave it
-    // for now as we're in the process of switching off Mailchimp (related: RW-705).
-    if (checkEmailVerification &&
```

Comment:
This parameter `checkEmailVerification` is no longer used, please remove.

Reply:
Done